source/Symbol/ClangASTContext.cpp

```diff
@@ -1437,8 +1437,11 @@ CompilerType ClangASTContext::CopyType(const CompilerType &src) {
           FileManager file_manager(file_system_options);
           ASTImporter importer(*dst_clang_ast, file_manager, *src_clang_ast,
                                file_manager, false);
-          QualType dst_qual_type(importer.Import(ClangUtil::GetQualType(src)));
-          return CompilerType(this, dst_qual_type.getAsOpaquePtr());
+          Expected<QualType> imported_dst_qual_type = importer.Import(ClangUtil::GetQualType(src));
+          if (imported_dst_qual_type) {
```

Comment:
I would fold this into one:

`if (auto T = importer.import(ClangUtil::GetQualType(src))`